rancher/v1.5/en/cattle/external-dns-service/index.md

```diff
@@ -15,6 +15,52 @@ As part of the [Rancher catalog]({{site.baseurl}}/rancher/{{page.version}}/{{pag
 * For every environment in your Rancher setup, there should be a `route53` service of scale 1.
 * Multiple Rancher instances should not share the same `hosted zone`.
 
+### Configuring IAM Permissions on AWS
+
+Unless you give Rancher permission, it will not be able to manage your Route53 zone file. 
+
```

Comment:
Instead of all these steps, we should just give the example policy that should be added. They should follow AWSs directions on how to add policies.